README.md

```diff
@@ -198,6 +198,7 @@ SUSHII_IMG_QUALITY=70
 
 * [sushii bot]
 * [haseul bot]
+* [hyejoo bot]
 * Use sushii-image-server? Feel free to open a PR to add to the list!
 
 [PM2]: https://github.com/Unitech/pm2
```

Comment:
GitHub isn't letting me give a suggestion at the very bottom, but should be easier to just add with this 😄 

```suggestion
[PM2]: https://github.com/Unitech/pm2
[hyejoo bot]: https://top.gg/bot/733035786664149104
```